README.md

```diff
@@ -7,3 +7,23 @@
 + Create and use variables
 + Use string methods
 + Use number methods
+
+
+## Instructions
+
```

Comment:
Add a fun intro - "Hooray, you're going shopping! First, you just have to build the store..."